ingenialink/enums/register.py

```diff
@@ -0,0 +1,55 @@
+from enum import Enum
+
+
+class REG_DTYPE(Enum):
+    """Data Type."""
+    U8 = 0
+    """Unsigned 8-bit integer."""
+    S8 = 1
+    """Signed 8-bit integer."""
+    U16 = 2
+    """Unsigned 16-bit integer."""
+    S16 = 3
+    """Signed 16-bit integer."""
+    U32 = 4
+    """Unsigned 32-bit integer."""
+    S32 = 5
+    """Signed 32-bit integer."""
+    U64 = 6
+    """Unsigned 64-bit integer."""
+    S64 = 7
+    """Signed 64-bit integer."""
+    FLOAT = 8
+    """Float."""
+    STR = 9
```

Comment:
In [ingenialink](https://github.com/ingeniamc/ingenialink/blob/master/include/public/ingenialink/registers.h#L39) the value of STR is 10

Reply:
It doesn't matter because when a register object is initialized, it depends on how this enum is defined, there is not anything that makes how to define the enum in a mandatory way.